mods/common/chrome/ingame-infostats.yaml

```diff
@@ -130,6 +130,16 @@ Container@SKIRMISH_STATS:
 							Checkmark: mute
 							Background: checkbox-toggle
 							TooltipContainer: TOOLTIP_CONTAINER
+						Button@KICK:
```

Comment:
Copy paste the `Button@KICK:` from line 175 to here. #20100 has changed how the button looks and works. 
Also do this in the other `ingame-infostats.yaml` files